frontend/Dockerfile

```diff
@@ -2,9 +2,6 @@ FROM registry.access.redhat.com/ubi8/nodejs-16:1-68.1666660386 as node
 
 WORKDIR /usr/app
 
-# Install npm version 8.5.5
-RUN npm i -g npm@8.5.5
```

Comment:
Beautiful

Reply:
Trying to get away from this. 

Yes its good practice to use a specific version of something, and not latest, but setting the npm version to something that isnt default in the node image just causes more technical debt and it wont get updated.